study.md

```diff
@@ -44,21 +48,25 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- view is a section of the HTML file
+The front -end routers save requests and avoids the browser from sending requests to the server when a page has been seen previously. -->
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+<!-- No installation required
+User experience is fluid  -->
```

Comment:
The second point is true, I'm not sure about the first being relevant/true though.